core/src/main/scala/filodb.core/store/ChunkSetInfo.scala

```diff
@@ -129,18 +130,26 @@ object ChunkSetInfo extends StrictLogging {
   /**
    * ChunkSetInfo metadata schema:
    * +0  long   chunkID
-   * +8  int    # rows in chunkset
-   * +12 long   start timestamp
-   * +20 long   end timestamp
-   * +28 long[] pointers to each vector
+   * +8  long   ingestion time (milliseconds from 1970)
+   * +16 int    # rows in chunkset
+   * +20 long   start timestamp
+   * +28 long   end timestamp
+   * +36 long[] pointers to each vector
    *
    * Note that block metadata has a 4-byte partition ID appended to the front also.
+   *
+   * Note 2: The chunkID is currently defined from the start timestamp and the ingestion time,
```

Comment:
We don't really need to store the start timestamp explicitly.... as it can be extracted completely from the chunkID right?   Why not just leave out the start timestamp field and derive it form the chunkID, then we save 8 bytes?

Reply:
I don't like the idea of hard-coding the chunkID TOO much. If the start timestamp is derived from the chunkID, there's no going back.

Reply:
I suppose..... though any change in the chunkID schema will be a breaking change anyways, it's not really any different from a chunkSetInfo schema change.

Reply:
Changed definition to closely match the current one. Less compatibility issues.